### PR TITLE
Save in Progress -- Save and show savedAt timestamp

### PR DESCRIPTION
**Pull request by cvalarida**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3197

Looks like https://marvelapp.com/2hj59b1/screen/28358733

Not logged in:
image

Logged in with saved form:
image

Logged in with no saved form:
image

### CODE REVIEW

**Comment by cehsu** [comment]
Do we want to include seconds in the timestamp?
screen shot 2017-06-13 at 9 57 42 am

**Comment by cvalarida** [comment]
Heh, I actually asked @stacyhlinn the same thing in @jbalboni's PR for the save redirect. I'll ask it again here!

What sayest thou, @stacyhlinn? To seconds or not to seconds? That is the question.

**Comment by stacyhlinn** [comment]
@cvalarida @cehsu Good question! 
Showing the seconds is probably not that valuable for the users, so let's just go with hour:minute am/pm. Thanks guys!

**Comment by ayaleloehr** [comment]
Pedantic questions: @stacyhlinn what time zone will that time be in and should we put that on there? (those are both actual questions I don't know the answers to, not leading questions!)

**Comment by stacyhlinn** [comment]
@ayaleloehr Thanks for bringing this up. And yes it should show the time in whichever timezone the users are in.  :)

**Comment by jbalboni** [comment]
@cvalarida You should still include a screenshot, even with the link to the marvel app. And also connect it to your GH issue.

File: src/sass/base/_b-utils.scss [hunk 1]
```diff
@@ -1,3 +1,7 @@
 .full-width {
   width: 100%;
 }
+
+.no-background-image {
```
Comment (jbalboni) [inline src/sass/base/_b-utils.scss]: Won't there still be left side padding here? I think making an alert specific no icon class would be good (i.e. a `usa-alert-no-icon` class that fixed the padding and hide the bg image).

**Review by jbalboni — APPROVED** [review]
Looks good to me. Can we add some tests for the intro page changes?

**Comment by cvalarida** [comment]
@jbalboni No, the padding comes from an inner `div.usa-alert-body`.

I'm having trouble getting this to work again (hoping just because of a change in the api), so I'm going to wait to get a screenshot out until [this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/5762) is merged. I'll then merge master into here and :crossed_fingers: hope it works again.

File: src/sass/base/_b-utils.scss (hunk 1 above)
Comment (cvalarida) [inline src/sass/base/_b-utils.scss]: No, the padding comes from an inner `div.usa-alert-body`.

**Review by cehsu — APPROVED** [review]

**Pull request merged**